services/net/contentmigration/ContentMigrationAction.cs

```diff
@@ -315,10 +315,35 @@ public class ContentMigrationAction : IngestAction<ContentMigrationOptions>
                 {
                     if (newsItem.FilePath != null)
                     {
-                        string contentStagingFolderName = GetOutputPathPrefix(manager.Ingest);
-                        await contentMigrator.CopyFileAsync(new Models.FileMigrationModel(newsItem.RSN, Path.GetDirectoryName(newsItem.FilePath)!, Path.GetFileName(newsItem.FilePath), newsItem.ContentType!), contentStagingFolderName);
-                        sourceContent.FilePath = Path.Combine(contentStagingFolderName, newsItem.FilePath);
-                        Logger.LogInformation("Migrating file associated with content content {RSN}:{Title}:[{filePath}]", newsItem.RSN, newsItem.Title, sourceContent.FilePath);
+                        if (string.IsNullOrEmpty(newsItem.ContentType)) {
+                            string ext = Path.GetExtension(newsItem.FilePath);
+                            switch (ext.ToUpper()){
+                                case ".JPEG":
+                                case ".JPG":
+                                    newsItem.ContentType = "image/jpeg";
+                                    break;
+                                case ".MOV":
+                                case ".MP4":
+                                case ".M4A":
+                                    newsItem.ContentType = "video/quicktime";
+                                    break;
+                                case ".PDF":
+                                    newsItem.ContentType = "application/pdf";
+                                    break;
+                                case ".TXT":
+                                    newsItem.ContentType = "text/plain";
+                                    break;
+                            }
+                        }
+
+                        if (string.IsNullOrEmpty(newsItem.ContentType)) {
```

Comment:
What other file types show up in the the DB?

Reply:
found two more after expanding my search criteria.